### ISSUE TITLE
spdlog/fmt/ranges.h No such file or directory

### ISSUE THREAD

**Issue by alphafox02** [gnuradio/gr-inspector#55]
Ubuntu 22.04 with libspdlog-dev installed, prior to this commit no issues. After this commit 

https://github.com/gnuradio/gr-inspector/commit/9abf5fbf11839debb96e7d9f8672017a6f2d8c5c

```
[ 48%] Building CXX object lib/CMakeFiles/gnuradio-inspector.dir/signal_marker.cc.o
[ 52%] Building CXX object lib/CMakeFiles/gnuradio-inspector.dir/ofdm_bouzegzi_c_impl.cc.o
/usr/src/gr-inspector/grinspector_new/lib/ofdm_bouzegzi_c_impl.cc:29:10: fatal error: spdlog/fmt/ranges.h: No such file or directory
   29 | #include <spdlog/fmt/ranges.h>
```

I notice in the above commit that the include was added.

**Comment by alphafox02** [comment]
Comment out the include and it builds, but I don't know enough about what it's doing/code changes to tell if it's really needed or not. I suspect since it compiles without the include that nothing in the .cc file is actually using it?

**Comment by alphafox02** [comment]
There's a /usr/include/fmt/ranges.h present, but not a spdlog/fmt/ranges.h that I can find in 22.04. Either way, it doesn't seem like the include is actually used/needed.

**Comment by marcusmueller** [comment]
Thanks, @alphafox02! Please review #56.

**Comment by marcusmueller** [comment]
solved by #56
